data/templates/ocserv/ocserv_passwd.tmpl

```diff
@@ -0,0 +1,6 @@
+#<username>:<group>:<hash>
+{% for user in username %}
```

Comment:
You should harden the for loop as keys now come and go: `{% for user in username if username is defined %}`

Reply:
I this is redundancy because this template will be rendered only if `username` defined
https://github.com/vyos/vyos-1x/pull/529/files/db55ef1b568d304b654deaa1646cde30a5913b99#diff-8f0bf8c31248481190ec5a459010e131R59